File: BE/KhoaLuanTotNghiep/app/Models/Booking.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Booking extends Model
{
    use HasFactory;
    protected $table = 'bookings';


    protected $fillable = [
        'customer_name',
        'email',
        'phone',
        'guests',
        'booking_date',
        'booking_time',
        'note',
        'status'
    ];
}

File: BE-springboot/src/main/java/com/quanlynh/entity/Booking.java
package com.quanlynh.entity;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Entity
@Table(name = "bookings")
@Getter
@Setter
public class Booking {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @ManyToOne
    @JoinColumn(name = "user_id")
    private User user;

    @Column(name = "customer_name", nullable = false)
    private String customerName;

    @Column(nullable = false)
    private String phone;

    @Column(name = "booking_time", nullable = false)
    private LocalDateTime bookingTime;

    private int guests;

    private String status;

    private String note;
}
